packages/react-components/src/components/Select/Select.css

```diff
@@ -115,18 +115,18 @@
 .bcds-react-aria-Select--Header {
   color: var(--typography-color-secondary);
   font: var(--typography-regular-small-body);
-  padding: var(--layout-margin-hair) var(--layout-padding-small);
+  padding: var(--layout-padding-hair) var(--layout-padding-small);
 }
 
 /* ListBox option item */
 .bcds-react-aria-Select--ListBoxItem {
-  border-radius: var(--layout-margin-xsmall);
+  border-radius: var(--layout-border-radius-medium);
   cursor: pointer;
   display: flex;
   flex-direction: row;
   align-items: center;
   gap: var(--layout-margin-small);
-  padding: 6.5px var(--layout-padding-small);
```

Comment:
I see this matches the current version of the component in Figma, but just flagging that this will cause the list of items to be physically taller:

Before:
<img width="1840" alt="Screenshot 2024-08-27 at 10 05 26 AM" src="https://github.com/user-attachments/assets/188ce8ef-10c0-491a-828d-7d43c216a881">

After:
<img width="1840" alt="Screenshot 2024-08-27 at 10 05 29 AM" src="https://github.com/user-attachments/assets/fbda0786-2e85-464c-a683-2609f8e3259d">

Reply:
Noted! I'm good with this if you are? My view is prioritising consistency between Figma/React here.

Reply:
Yep, I agree with your priority there, no changes!